packages/graphql-language-service-server/src/MessageProcessor.ts

```diff
@@ -664,42 +664,47 @@ export class MessageProcessor {
           await this._updateFragmentDefinition(uri, contents);
           await this._updateObjectTypeDefinition(uri, contents);
 
-          const project = this._graphQLCache.getProjectForFile(uri);
-          if (project) {
-            await this._updateSchemaIfChanged(project, uri);
-          }
+          try {
+            const project = this._graphQLCache.getProjectForFile(uri);
+            if (project) {
+              await this._updateSchemaIfChanged(project, uri);
+            }
 
-          let diagnostics: Diagnostic[] = [];
-
-          if (
-            project?.extensions?.languageService?.enableValidation !== false
-          ) {
-            diagnostics = (
-              await Promise.all(
-                contents.map(async ({ query, range }) => {
-                  const results = await this._languageService.getDiagnostics(
-                    query,
-                    uri,
-                    this._isRelayCompatMode(query),
-                  );
-                  if (results && results.length > 0) {
-                    return processDiagnosticsMessage(results, query, range);
-                  }
-                  return [];
-                }),
-              )
-            ).reduce((left, right) => left.concat(right), diagnostics);
-          }
+            let diagnostics: Diagnostic[] = [];
+
+            if (
+              project?.extensions?.languageService?.enableValidation !== false
+            ) {
+              diagnostics = (
+                await Promise.all(
+                  contents.map(async ({ query, range }) => {
+                    const results = await this._languageService.getDiagnostics(
+                      query,
+                      uri,
+                      this._isRelayCompatMode(query),
+                    );
+                    if (results && results.length > 0) {
+                      return processDiagnosticsMessage(results, query, range);
+                    }
+                    return [];
+                  }),
+                )
+              ).reduce((left, right) => left.concat(right), diagnostics);
+            }
 
-          this._logger.log(
-            JSON.stringify({
-              type: 'usage',
-              messageType: 'workspace/didChangeWatchedFiles',
-              projectName: project?.name,
-              fileName: uri,
-            }),
-          );
-          return { uri, diagnostics };
+            this._logger.log(
+              JSON.stringify({
+                type: 'usage',
+                messageType: 'workspace/didChangeWatchedFiles',
+                projectName: project?.name,
+                fileName: uri,
+              }),
+            );
+            return { uri, diagnostics };
+          } catch (err) {
+            this._handleConfigError({ err, uri });
+            return { uri, diagnostics: [] };
+          }
```

Comment:
You might want to reformat as per the standards in this repository. May be try `npx prettier --write .`

Reply:
Hi @tusharmath, there was a pre-commit hook running `prettier`. So, I did not run it manually. I verified just now and these changes are left as is.

Reply:
`yarn format` is what you want, so that eslint fix applies as well

Reply:
Ah, ok. There were no changes after running `yarn format` too, @acao. Can this be merged or do you want me to make any changes? Thanks.

Reply:
@A-N-uraag I will take a look this week when I have time, trying to find a new job currently haha

Reply:
Thanks for taking the time and merging it, @acao. Good luck on your new job search.